src/odb/include/odb/db.h

```diff
@@ -1177,6 +1177,26 @@ class dbBlock : public dbObject
   ///
   int getDbUnitsPerMicron();
 
+  ///
+  /// Convert a length from database units (DBUs) to microns.
+  ///
+  double dbuToMicrons(const int64_t dbu);
```

Comment:
I think dbu is a regular 32-bit int? It's only areas that are 64.

Reply:
Correct! Further information: https://openroad.readthedocs.io/en/latest/contrib/DatabaseMath.html